src/endpoints/uploadAndCloudinary.ts

```diff
@@ -0,0 +1,50 @@
+import express from "express";
+import getRawBody from "raw-body";
+import { actions } from "../actions";
+import { FleekUploadedFile } from "../storage";
+import { Maybe } from "../types";
+import { convertToResponse, responses } from "../utilities";
+
+export const uploadAndCloudinary = async (
+  req: express.Request,
+  res: express.Response,
+  next: express.NextFunction
+) => {
+  try {
+    let contents: Maybe<Buffer | string>;
+
+    if (req.readable) {
+      try {
+        contents = await getRawBody(req, {
+          length: req.headers["content-length"],
+          limit: "100mb",
+        });
+      } catch (e) {
+        if (e.type === "entity.too.large") {
+          res.status(413);
+          throw Error(
+            `File too large, limit is ${(e.limit / 1024 / 1024).toPrecision(
```

Comment:
consider moving this small calculation to its own line so that the string interpolation can be a single line as well for readability

Reply:
Moved to a helper